src/modules/auth/ws-auth.guard.ts

```
@@ -0,0 +1,39 @@
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+      const payload = await this.jwtService.verifyAsync<JwtPayload>(token);
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+      client.data = { user: payload };
+    } catch {
+      throw new WsException('Unauthorized');
+    }
+
+    return true;
+  }
+
+  private extractTokenFromHandshake(client: SocketWithUser): string | undefined {
```

ESLint

[error] Replace `client:·SocketWithUser` with `⏎····client:·SocketWithUser,⏎··`

(prettier/prettier)